packages/react-components/src/components/table-output-component.tsx

```diff
@@ -107,7 +112,12 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
             if (timestampCol) {
                 const timestamp = event.data[timestampCol];
                 const payload = { 'timestamp': timestamp };
-                this.lastIndex = { timestamp: Math.trunc(Number(timestamp)), index: event.rowIndex };
+                this.lastIndex = {
+                    startTimestamp: Math.trunc(Number(timestamp)),
+                    endTimestamp: Math.trunc((Number(timestamp))),
+                    startIndex: event.rowIndex,
+                    endIndex: event.rowIndex
+                };
```

Comment:
right now only the start timestamp is broadcasted below (fireSelectionChangedSignal). If user selects multiple rows, the you need to broadcast the start and end time of the range so that other views (e.g. timeline-chart) can draw the selection change)